README.md

```diff
@@ -75,6 +75,27 @@ Rename and edit the existing permissions file, changing both `name` and the last
 
 Change the `paths` to match the new Maven coordinates, or, if further uploads for the old coordinates are expected, add a new list entry.
 
+Managing Continuous Delivery (JEP-229 CD)
+-----------------------------------------
+
+Jenkins plugins and other components can be continuously delivered through a supported process described in [JEP-229](https://github.com/jenkinsci/jep/blob/master/jep/229/README.adoc).
+
+You can enable JEP-229 CD for your component by adding the following to your component's YAML file:
+
+```yaml
+cd:
+  enabled: true
+```
+
+**NOTE:** As of July 2022, this does not work for multimodule projects deploying multiple components, as these would have multiple YAML files in this repository.
```

Comment:
Really? I am pretty sure that https://github.com/jenkinsci/pipeline-model-definition-plugin gets released via CD. See https://github.com/jenkins-infra/repository-permissions-updater/pull/2324.

Reply:
as does JCasC and a few others

Reply:
Huh, interesting. I didn't think that worked. #1747 PR comment indicates it wouldn't:

> In fact, the first item above is made worse here, this might create tokens that can only upload a subset of artifacts in a repo.

That doesn't seem to be the case looking at the code, as the per-repo group (to which the per-repo token is added) is part of all individual permission targets for each listed artifact. I guess I might have considered the edge case when not every single artifact in a GH repo declares the correct and identical `github` value? I don't remember.

Anyway, I'm happy to delete this (and even happier to learn that past me might actually have considered this problem).

```suggestion
```